test/dynamo/test_misc.py

```diff
@@ -4409,6 +4410,7 @@ def foo(x):
         del x
         self.assertIs(x_ref(), None)
 
+    @xfailIfSingleStepGraph
```

Comment:
is this really an xfail, or is this test just totally inapplicable haha

Reply:
I assume when we set the `use_single_step_graph` to False it will pass... Through I don't know if we actually want to keep this test.

Reply:
If the test doesn't make any sense with use single step graph, it's better to just skip it when single step as opposed to xfailing it.

Reply:
sg, will do